packages/app/src/app/components/CreateNewSandbox/CreateSandbox/elements.ts

```diff
@@ -186,3 +186,31 @@ export const TemplateGrid = styled.div`
   grid-template-columns: repeat(3, 1fr);
   gap: 10px;
 `;
+
+export const SelectContainer = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: row;
+`;
+
+export const inputStyles = css`
+  height: 32px;
```

Comment:
can you reference some of the tokens here? or is it a specific style for the input that can't rely on the tokens? talking about the css builder from @styled-system/css